### DISCUSSION TITLE
Purpose of Framework Master Category

### DISCUSSION THREAD

**Discussion by coolbung** [Sunbird-Knowlg/Community#123] [Q&A] [answered]
Wish to understand the purpose of the Framework Master Categories. 

Here are the steps to be followed to create a Framework
1. Create Framework Master Categories
2. Create Framework
3. Create Framework Categories (i.e. map Master Categories to Framework)
4. Create Terms for each Framework Category (not Master Category)
5. Optionally, create associations between Terms

Wanted to understand what was the intention behind having Framework Master Categories, since
1. There is no way to reuse Terms
2. When the framework is representing different domains (eg: education vs. agriculture vs. FRAC vs. software programming), different master categories need to be created for each framework
3. Even in case where multiple Frameworks are using the same Master Category, defining the Master Category does not seem to serve a significant purpose, since the Categories are again created for the Framework.

@surendrasinghs @vrayulu @Krishnaj20 @pramodkvarma 

cc @vinod303 @parthlawate @manojlondhe

**Comment by vrayulu** [comment, marked as answer]
Master framework categories are to avoid creating duplicate categories with different names (e.g. gradeLevel, grade, class, etc) and to know the complete list of categories across all frameworks. This also ensures review before a new framework category is added.

In addition to the above: if a vocabulary layer is created where terms can be related with each other, the master categories can be used to query for equivalent terms (or by other relation) within the same category (i.e. scoping on terms within a category becomes possible).

**Reply by pallakartheekreddy** [reply]
@coolbung Hope your query is now resolved. Please mark the thread as answered
